docs/Configurations.md

```diff
@@ -1,99 +1,86 @@
 # FCA: Configurations
 
-TODO: add intro
+This document gives the various configurations available for Firebolt Certification App (FCA). It provides insights into setting up and utiizing different environments, parameters, and app types to facilitate effective testing and certification of applications using firebolt technology. 
+
 
 ## Table of Contents
 
-- [Supported Targets](#supported-targets)
+- [Running with MFOS](#running-with-mfos)
 - [Connect to Mock Firebolt](#connect-to-mock-firebolt)
 - [Supported URL Parameters](#supported-url-parameters)
 - [App Types](#app-types)
 
-## Supported targets
-
-EXISTING CONTENTS(needs revision):
-> - MFOS - `https://github.com/rdkcentral/mock-firebolt`
+## Running with MFOS
 
 ### Background
 
-TODO: add details
+Mock Firebolt (MFOS) is a simulated environment that FCA can interact with.
```

Comment:
Think we can add more details to explain to the user, what is MFOS simulating? The device the firmware, the APIs?

Reply:
Very abbreviated description of what MFOS is and does would be good and we can link to MFOS documentation guide for more info.

Reply:
@jbigel  link to documentation is linked in the setup, should I move it from there ?

Reply:
updated to mention what a simulated environment means in terms of MFOS :
Mock Firebolt (MFOS) is a simulated environment representing (real Firebolt running on device) that FCA can interact with.
also updated usage to :
Utilize MFOS for testing in a controlled, simulated environment. MFOS can be ideal for early-stage development and testing.
Use MFOS if you don't have a device with firebolt running on it, or don't have a device at all.

Reply:
`representing` should be inside the `(`